src/util/index.ts

```diff
@@ -76,3 +76,7 @@ export const wrapText = (
 export const clearUrlParams = () => {
     history.replaceState({}, "", `${location.origin}${location.pathname}`);
 };
+
+export const roundTimeForDisplay = (time: number): number => {
+    return parseFloat(time.toPrecision(3));
+};
```

Comment:
i think we said in the test meeting that we want to cover utils too. It's a super simple function to test, but could be important incase we ever change the precision expectations

Reply:
Done!